build_runner/lib/src/asset/finalized_reader.dart

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
```

Comment:
[nit] extra blank line below the copyright to separate it from imports

Reply:
Done here and elsewhere.